src/components/members/memberList.tsx

```
@@ -1,139 +1,654 @@
+        </Text>
+      </Paper>
+      
+      {filteredUsers?.length === 0 ? (
+        <Center p={50}>
+          <Box ta="center">
+            <ThemeIcon size={60} radius="xl" mb="md">
+              <IconSearch size={30} stroke={1.5} />
+            </ThemeIcon>
+            <Text fw={700} size="lg">No members found</Text>
+            <Text c="dimmed" size="sm" maw={400} mx="auto" mt="xs">
+              Try adjusting your search or filter criteria to find what you're looking for.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)